indico/client/request.py

```diff
@@ -90,13 +90,13 @@ def requests(self):
 
 
 class Debouncer:
-    def __init__(self, max_timeout: int = 5):
+    def __init__(self, max_timeout: Tuple[int, float] = 5):
```

Comment:
Added kwarg `max_wait_time` to all client calls that use `Debouncer` to allow access to change this variable.